todo_app/app.py

```diff
@@ -1,23 +1,26 @@
 from flask import Flask, render_template, request, redirect
+from todo_app.ViewModel import ViewModel
 from todo_app.flask_config import Config
-from todo_app.data.trello_items import get_done_items, get_to_do_items, add_item, complete_item
+from todo_app.data.trello_items import get_to_do_items, get_doing_items, get_done_items, add_item, complete_item
 
-app = Flask(__name__)
-app.config.from_object(Config())
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object(Config())
 
-@app.route('/')
-def index():
-    to_do_items = get_to_do_items()
-    done_items = get_done_items()
-    return render_template('index.html', items=to_do_items+done_items)
+    @app.route('/')
+    def index():
+        item_view_model = ViewModel(get_to_do_items(), get_doing_items(), get_done_items())
+        return render_template('index.html', view_model=item_view_model)
 
-@app.route('/submit', methods=['POST'])
-def addItem():
-    newItem = request.form.get('title')
-    add_item(newItem)
-    return redirect('/')
+    @app.route('/submit', methods=['POST'])
+    def addItem():
+        newItem = request.form.get('title')
+        add_item(newItem)
+        return redirect('/')
 
-@app.route('/completeItem<id>')
-def complete_item_route(id):
-    complete_item(id)
-    return redirect('/')
\ No newline at end of file
+    @app.route('/completeItem<id>')
```

Comment:
It'd be slightly nicer for this to be a POST endpoint since it modifies data, though it means your HTML has to be slightly more complicated (a form with a submit instead of just an anchor)

And also it'd be nicer to have a slash between "completeItem" and the id.